backend/de.metas.contracts/src/main/java/de/metas/contracts/FlatrateTermRequest/ModularFlatrateTermQuery.java

```diff
@@ -49,7 +49,7 @@ public class ModularFlatrateTermQuery
 	@Nullable
 	CalendarId calendarId;
 	@Nullable
-	Timestamp dateFrom;
+	Timestamp dateFromLessOrEqual;
 	@Nullable
-	Timestamp dateTo;
+	Timestamp dateToGreaterOrEqual;
```

Comment:
now or on next issue, pls consider converting those Timestamp fields to Instant...